odoo/tests/common.py

```diff
@@ -1441,7 +1441,7 @@ def start_tour(self, url_path, tour_name, step_delay=None, **kwargs):
         `browser_js` can be passed as keyword arguments."""
         step_delay = ', %s' % step_delay if step_delay else ''
         code = kwargs.pop('code', "odoo.startTour('%s'%s)" % (tour_name, step_delay))
-        ready = kwargs.pop('ready', "odoo.__DEBUG__.services['web_tour.tour'].tours['%s'].ready" % tour_name)
+        ready = kwargs.pop('ready', "odoo?.__DEBUG__?.services?.['web_tour.tour']?.tours?.['%s']?.ready" % tour_name)
```

Comment:
Might be a better idea to put the entire `ready` expression in a try/except and return `False` as long as the evaluation fails.

That seems even more relevant as the `ready` parameter is overridable, so the same issue could theoretically occur on tours with a bespoke `ready` expression. 

In fact it might be a good idea to lift that into `browser_js`, as `start_tour`'s `ready` doesn't seem to be used currently but `browser_js`'s is.